src/main/java/htsjdk/samtools/util/CigarUtil.java

```diff
@@ -31,17 +31,20 @@
 import htsjdk.samtools.SAMValidationError;
 import htsjdk.samtools.TextCigarCodec;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.LinkedList;
-import java.util.List;
+import java.util.*;
 
 /**
  * @author alecw@broadinstitute.org
  */
 public class CigarUtil {
         private static final Log log = Log.getInstance(CigarUtil.class);
 
+
+    public enum Clipping {
```

Comment:
I lean toward using `CigarOperator`, which contains these, rather than introducing a new enum.  Your method could then throw an exception if someone tried to use a non-clipping operator.